packages/image_picker/image_picker/example/ios/RunnerUITests/ImagePickerFromLimitedGalleryUITests.m

```diff
@@ -25,31 +25,18 @@ - (void)setUp {
   __weak typeof(self) weakSelf = self;
   [self addUIInterruptionMonitorWithDescription:@"Permission popups"
                                         handler:^BOOL(XCUIElement* _Nonnull interruptingElement) {
-                                          if (@available(iOS 14, *)) {
-                                            XCUIElement* limitedPhotoPermission =
-                                                [interruptingElement.buttons elementBoundByIndex:0];
-                                            if (![limitedPhotoPermission
-                                                    waitForExistenceWithTimeout:
-                                                        kLimitedElementWaitingTime]) {
-                                              os_log_error(OS_LOG_DEFAULT, "%@",
-                                                           weakSelf.app.debugDescription);
-                                              XCTFail(@"Failed due to not able to find "
-                                                      @"selectPhotos butt   on with %@ seconds",
-                                                      @(kLimitedElementWaitingTime));
-                                            }
-                                            [limitedPhotoPermission tap];
-                                          } else {
-                                            XCUIElement* ok = interruptingElement.buttons[@"OK"];
-                                            if (![ok waitForExistenceWithTimeout:
-                                                         kLimitedElementWaitingTime]) {
-                                              os_log_error(OS_LOG_DEFAULT, "%@",
-                                                           weakSelf.app.debugDescription);
-                                              XCTFail(@"Failed due to not able to find ok button "
-                                                      @"with %@ seconds",
-                                                      @(kLimitedElementWaitingTime));
-                                            }
-                                            [ok tap];
+                                          XCUIElement* limitedPhotoPermission =
+                                              [interruptingElement.buttons elementBoundByIndex:0];
+                                          if (![limitedPhotoPermission
+                                                  waitForExistenceWithTimeout:
+                                                      kLimitedElementWaitingTime]) {
+                                            os_log_error(OS_LOG_DEFAULT, "%@",
+                                                         weakSelf.app.debugDescription);
+                                            XCTFail(@"Failed due to not able to find "
+                                                    @"selectPhotos butt   on with %@ seconds",
```

Comment:
Could you fix the type while you are here? ("butt   on with" -> "button within")